convert2rhel/unit_tests/toolopts_test.py

```diff
@@ -145,3 +146,50 @@ def test_both_disable_submgr_and_no_rhsm_options_work(argv, raise_exception, no_
         convert2rhel.toolopts.CLI()
 
     assert convert2rhel.toolopts.tool_opts.no_rhsm == no_rhsm_value
+
+
+@pytest.mark.parametrize(
```

Comment:
Maybe too much tests, but I want to show what happens in different cases.  Believe it will be useful for reviewers and makes your work easier :)

Reply:
I like how you have all of these test cases!  Thanks!